src/main/java/org/apache/mesos/specification/Named.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.mesos.specification;
+
+/**
+ * A named object provides a name.
```

Comment:
This feels so abstract that I think we could just leave it out. Will anyone need to have a collection of different types of `Named`s?